mokapot/model.py

```diff
@@ -421,14 +446,15 @@ def __init__(
         override=False,
         subset_max_train=None,
         n_jobs=-1,
+        supervised_step=False,
     ):
         """Initialize a PercolatorModel"""
         self.n_jobs = n_jobs
         svm_model = LinearSVC(dual=False)
         estimator = GridSearchCV(
             svm_model,
             param_grid=PERC_GRID,
-            refit=False,
+            refit=True,
```

Comment:
Why do we need to add this?

Reply:
Right, we don't. This is redundant since we are manually refitting the best parameters in _find_hyperparameters()